SU-SWS/stanford_paragraph_types
# Hsdo 771 icon grid

# READY FOR REVIEW

# Summary
- Added a icon grid paragraph item similar to the icon grid from stanford_bean_types

# Needed By (Date)
- 4/12

# Steps to Test

1. Checkout Branch
2. Enable stanford_paragraph_types & stanford_story_page, clear cache/revert feature
3. add a new stanford story page content
4. Add the paragraph item 'Icon Grid'
5. populate with font awesome icons & transparent png icons (50x50 to 100x100 sizes)
6. create 1 to 50. Should produce rows of 3 centered.

# Notes
requires PR branch on [stanford_field_formatters](https://github.com/SU-SWS/stanford_field_formatters/pull/2)
